app/uk/gov/hmrc/cdsreimbursementclaimfrontend/views/claims/check_claim.scala.html

```diff
@@ -18,23 +18,17 @@
 @import play.api.i18n.Messages
 @import uk.gov.hmrc.cdsreimbursementclaimfrontend.config.ViewConfig
 @import uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.actions.RequestWithSessionData
-@import  uk.gov.hmrc.cdsreimbursementclaimfrontend.models.Claim
 @import  uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.claims.routes
-@import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryList
-@import cats.data.NonEmptyList
 
 @this(
  layout: uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.Layout,
- claimSummaryHelper: uk.gov.hmrc.cdsreimbursementclaimfrontend.views.utils.ClaimSummaryHelper,
  submitButton: uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.components.button,
  pageHeading: uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.components.page_heading,
-formWithCSRF: uk.gov.hmrc.govukfrontend.views.html.helpers.formWithCSRF,
-
-summary: uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.components.summary,
+ formWithCSRF: uk.gov.hmrc.govukfrontend.views.html.helpers.formWithCSRF,
  paragraph: uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.components.paragraph_block
 )
 
-@(claims : NonEmptyList[Claim])(implicit  request: RequestWithSessionData[_], messages:Messages, viewConfig: ViewConfig)
+@()(implicit  request: RequestWithSessionData[_], messages:Messages, viewConfig: ViewConfig)
 
     @key = @{"check-claim"}
```

Comment:
this should be `check-claim-summary` as that's what you call set the message key to be in the scala form model - this keeps it inline with the convention we have in the service